.github/actions/cargo-semver-checks/action.yml

```
@@ -17,10 +17,15 @@ runs:
         MAX_STABLE_VERSION=$(curl https://crates.io/api/v1/crates/${{ inputs.crate }} --silent | jq '.crate.max_stable_version')
         echo "version=${MAX_STABLE_VERSION}" >> $GITHUB_OUTPUT
 
+    - shell: bash
+      run: |
+        rustc --version | tee .rustc-version
+        cargo semver-checks | tee .semver-checks-version
```

It's probable that in a future release, this will become equivalent to `cargo semver-checks check-release`: https://github.com/obi1kenobi/cargo-semver-checks/issues/414

Let me quickly add an explicit `--version` option instead, so I don't break your code whenever that change is shipped.

---

Ah there's actually already a `--version` option thanks to clap magic!

```suggestion
        cargo semver-checks --version | tee .semver-checks-version
```

---

That is what I wanted actually, just forgot to type it!